### DISCUSSION TITLE
Don't have access to my own projects

### DISCUSSION THREAD

**Discussion by Toderdenemn** [AntlerVC/firetable#1245] [Support / Q&A] [answered]
i buy pro version but don't have access to my own projects!!

**Comment by bofeiw** [comment, marked as answer]
Sorry for your inconvenience. 

If you are having access issue during project setup, please check if this helps:
https://docs.rowy.io/trouble-shooting/project-creation
We just discovered the issue that project setup fails but continues to redirect you to project page. We will push out a fix soon.

If you have other access issues, please describe your issues and we will help you.
